### ISSUE TITLE
Make topos more discoverable

### ISSUE THREAD

**Issue by d21d3q** [theCrag/website#4227]
Great project, but I'm struggling to use it as topo. If I am browsing map to find some climbing area I can see that there are many routes added, but I can't distinguish if there is any topo for them. Sometimes there is area that is partially with topo. But one needs to click through lots of links to find if there is anything usefull.  
Logging ascent is secondary feature (for now I am using 8a.nu to log ascents, but at home after climbing)

Proposed solutions (features):
* add filtering to the map which will show only routes that have topo
* make alternative frontend that will be topo focused - without non-topo features, lightweight (so that it works on mobile with bad connection in the field).

**Comment by lordyavin** [comment]
Hey @d21d3q did you try to search for topos in the area you want to climb? E.g. https://www.thecrag.com/en/climbing/switzerland/alpen/appenzeller-alpen/area/157688322/topos. Is such a list what you need? If you need something that work better on mobiles check out the apps that integrate date from theCrag.

**Comment by georg-d** [comment]
> Hey @d21d3q did you try to search for topos in the area you want to climb? 

For me, this is of little help because often, several "climbing areas" are around a point of interest. Because everyone does a slightly different definition of "climbing areas" _and_ they usually have multiple names (e.g. dialects/languages) _and_ they exist in several hierarchy levels _and_ they can be partly overlapping, I simply don't know the exact spelling and border of all relevant areas by heart. Hence, I need to see them on the map – and I don't see how can I can create a collection of areas on the map and forward this collection to the topo search. 
Example: Give me all crags 1h around Locarno [in real life: but only crags that have a topo and offer sufficient amount of routes to fill a day; see below]. Map https://www.thecrag.com/climbing/switzerland/alpen/tessin/maps#46.1834,8.7077,11.0,,terrain.maptiler Either I must manually loop through a lengthy list of areas, or I take "Ticino" and also get crags much too distant while nearby crags located in Italy do not show up. This becomes even worse where 3 or 4 countries are around.

**Comment by georg-d** [comment]
I fully agree to the base observation and for sake of wider benefit, I'd like to generalize the feature request a bit to "allow to add _several_ filters for the map", e.g.

- crags with topos or without (for those willing to contribute) or "I don't care" (e.g. in areas where route names are written on the wall so a topo is not needed but a pure route listing is sufficient)
- crags with certain tags, e.g. only south exposition without trees
- crags with a certain amount of routes in a certain grades range, e.g. at least 5 routes 4a to 5a so family members can also have fun

Why not use the search function? Because it returns a list. This neither contains any distance information (same example use case as above, i.e. "1 hour around my Locarno") nor does it show the topography – and climbing often happens in mountainous regions where information on mountain ranges & valleys are heavily important for estimation of wind, temperature, sun/shadow, snow, approach time etc.

From implementation and UX point of view, I imagine it to be quite easy: Add an icon with gears or 3-line. Clicking it opens a pop up. Within that popup, users can define the same filters as in current _advanced search_ for areas or topos, and return to the map where filtering applies. Of course, the new usage of search/filter may imply some re-work of current search, e.g. that areas can also be searched for _"how many routes it contains from min to max grade"_ – this filter logic already exists, it just needs to be also exposed for areas search (not only routes) and the results need to be counted (instead listed).
